### DISCUSSION TITLE
What is about a meetup in Bay Area (San Francisco, Sunnyvale)

### DISCUSSION THREAD

**Discussion by rtsisyk** [organicmaps/organicmaps#9443] [Ideas]
Would anyone like to meet in the Bay Area (San Francisco, Sunnyvale, Mountain View, etc.) to discuss Organic Maps, OpenStreetMap, and open-source in general? I will be around in the next two weeks.

**Comment by VeryUsual** [comment]
Limiting it to a physical region lowers the amount of people who can join, so I suggest maybe an **online** meetup would be more popular.
